stack.yaml

```diff
@@ -1,6 +1,6 @@
 # Configures GHC to be the 8.2 variant from the Docker image
 # See https://hub.docker.com/r/tweag/linear-types/
-resolver: ghc-8.2
+resolver: nightly-2017-12-20
```

Comment:
Can use LTS-10 now.

Reply:
Ha, I merged this in a separate PR before you brought it up. Well… no worries, I'll fix it in the afternoon.